jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/DumpableAttributes.java

```diff
@@ -0,0 +1,46 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.util.component;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.eclipse.jetty.util.Attributes;
+
+public class DumpableAttributes implements Dumpable
+{
+    private final String _name;
+    private final Attributes _attributes;
+
+    public DumpableAttributes(String name, Attributes attributes)
+    {
+        _name = name;
+        _attributes = attributes;
+    }
+
+    @Override
+    public String dump()
+    {
+        return Dumpable.dump(this);
```

Comment:
Isn't this the default implementation?

Reply:
yep. Copied from DumpableCollection, so fixing there too.